ground/src/main/java/com/google/android/ground/persistence/local/room/converter/LoiPropertiesMapConverter.kt

```diff
@@ -0,0 +1,45 @@
+package com.google.android.ground.persistence.local.room.converter
```

Comment:
Please add copyright header to new files

Reply:
Done